src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,28 @@
+.calendar {
+  padding: 10px;
+  width: #{$limit-width-of-row};
+  display: flex;
+  gap: $indent-of-days;
+  flex-wrap: wrap;
+
+  &__day {
+    box-sizing: border-box;
+    text-decoration: none;
+
+    width: $day-size;
+    height: $day-size;
+    background-color: #eee;
+    border: 1px solid black;
+
+    align-items: center;
+    justify-content: center;
+    transition: 500ms;
+    display: none;
+  }
+
+  &__day:hover {
+    cursor: pointer;
+    background-color: #ffbfcb;
+    transform: translateY(-20px);
```

Comment:
You have set a hover effect on the `&__day` element which changes the `background-color` and applies a `transform`. However, since the `&__day` elements are not visible due to `display: none;`, this effect will not be visible. You need to make sure that the elements you want to apply the hover effect to are visible.